gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineVersionsIntegrationTest.groovy

```diff
@@ -151,6 +151,30 @@ class BaselineVersionsIntegrationTest  extends AbstractPluginTest {
 
     }
 
+    def 'Version props conflict without exact artifact should fail'() {
+        when:
+        setupVersionsProps("com.fasterxml.jackson.*:* = 2.9.6\ncom.fasterxml.jackson.core:* = 2.9.5")
+        buildFile << standardBuildFile(projectDir) + """
+        dependencies {
+            compile 'com.fasterxml.jackson.core:jackson-core'
+        }"""
+
+        then:
+        buildAndFailWith("")
+    }
+
+    def 'Version props conflict with exact artifact should succeed'() {
+        when:
+        setupVersionsProps("com.fasterxml.jackson.*:* = 2.9.6\ncom.fasterxml.jackson.core:jackson-core = 2.9.5")
```

Comment:
what happens if you have `com.fasterxml.jackson.core:jackson-core = 2.9.6\ncom.fasterxml.jackson.core:jackson-core = 2.9.5"`

Reply:
With the solution proposed above, that wouldn't fail but it probably should. We should likely fail on conflicts that are specialized.